### DISCUSSION TITLE
Introduction to GitHub course

### DISCUSSION THREAD

**Discussion by danuja-illeperuma** [skills/.github#255] [Ideas]
Very good course. it was very helpful for beginners.

**Comment by Vue-Pu** [comment]
Yes, a quick Push to start on the right foot!

**Comment by shreyash-wontdoit** [comment]
ypu guys completed "Communicate using markdown"? would appreciate help

**Reply by Vue-Pu** [reply]
Good Daywhat kind of help are you seeking or step of the course are you finding it difficult, #900 ?

**Comment by eemden777** [comment]
il est ou le cours ?
